src/utils/misc.utils.js

```diff
@@ -0,0 +1,20 @@
+import { axiosInstance } from "../modules/sign/components/axiosApi"
+
+export const setGitLabAuth = () => {
+    localStorage.setItem('auth', "gitlab")
+    return true
+}
+
+export const SetGitHubAuth = () => {
+    localStorage.setItem('auth', "github")
```

Comment:
What is the use of this flag being set in localStorage?

Reply:
So as to persist the value of the isGitlab state property. Because the page refreshes after the Oauth sends back the "code" token, and  it sets the isGitlab property back to it's default state value "false", which affects the request being sent to the backend server endpoint.

Reply:
@ArinzeJeffrey-droid do you think this information could be added to the redirect URL?

Reply:
@toystars Not sure, But I'll look into it. Thanks.